rosie.py

```diff
@@ -1,4 +1,20 @@
+"""Rosie.
+
+Usage:
+    python rosie.py run (chamber_of_deputies | federal_senate)
+    python rosie.py run (chamber_of_deputies | federal_senate) [--years 2017,2016]
+    python rosie.py run (chamber_of_deputies | federal_senate) [--path <path to output directory>]
+    python rosie.py run (chamber_of_deputies | federal_senate) [--path <path to output directory>] [--years 2017,2016]
+    python rosie.py test [chamber_of_deputies | federal_senate]
+
+Options:
+    --years   runs rosie wuth years filter
+    --path    runs rosie with specific directory
```

Comment:
Rosie is a proper noun and should be capitalized.

Also runs _runs rosie with specific directory_ doesn't tell much about what this directory is for What about: _directory in which Rosie saves fetched data and the suspicions CSV_?

If I'm not wrong this optional argument has a default (`data/`? `/tmp/serenata-data/`?) that should be documented as well.

Reply:
> If I'm not wrong this optional argument has a default (data/? /tmp/serenata-data/?) that should be documented as well.

Yes, `serenata-data`, can you help me documenting that? There is something in the `README.md` but I guess we can do more.

Reply:
>  can you help me documenting that?

From [the snippet of `docopt` docs I pasted yesterday](https://github.com/datasciencebr/rosie/pull/115#discussion_r159600820) it's is as simple as replacing this very line we're commenting on. Just change:

```
    --path    runs rosie with specific directory
```

by:


```
    --path    runs rosie with specific directory [default: /tmp/serenata-data]
```

Reply:
accepted